SRC/classes/ParentChildSample.cls

```diff
@@ -0,0 +1,52 @@
+/**
+*  Description     :  Insert Parent Child Record using foreign key
+*
+*  Created By      :  Kamlesh Kumar
+*
+*  Created Date    :  07/11/2022
+*
+*  Revision Logs   :    V1.0 - Created - Kamlesh Kumar
+**/
+public with sharing class ParentChildSample {
```

Comment:
From Next time follow proper naming convention **what is mean of adding Sample you write example **